crypto/bio/b_addr.c

```diff
@@ -207,7 +207,8 @@ static int addr_strings(const BIO_ADDR *ap, int numeric,
                                flags)) != 0) {
 # ifdef EAI_SYSTEM
             if (ret == EAI_SYSTEM) {
-                FUNCerr("getnameinfo", get_last_socket_error());
+                ERR_raise_data(ERR_LIB_SYS, get_last_socket_error(),
+                               "calling %s()", "getnameinfo");
```

Comment:
I noticed you do this for all functions where you don't present any parameters... But how about this instead?

``` C
                ERR_raise_data(ERR_LIB_SYS, get_last_socket_error(),
                               "calling getnameinfo()");
```

Reply:
I wanted to get into the "safe" habit of always having a format string, so that nobody would slip in a ```%s``` (by mistake or malice).  Should I not bother?

Reply:
I don't see a reason to bother, all you really do is to make it harder to read.

Reply:
Okay, changed the "%s" to have the inline function name.

Reply:
The original version would allow a compiler/linker to merge all the `calling %s()` strings and save a bit of space.  This can be useful on smaller embedded systems.  Not a big deal though.